src/bindings/js/node/tests/model.test.js

```diff
@@ -28,3 +28,21 @@ describe('Node.js Model.isDynamic()', () => {
     assert.strictEqual(model.isDynamic(), expectedStatus, 'Expected isDynamic to return false for a static model');
   });
 });
+
+describe('Node.js Model.getOutputSize()', () => {
```

Comment:
'Node.js' сan be omitted

```suggestion
describe('Model.getOutputSize()', () => {
```